hnn_core/extracellular.py

```diff
@@ -233,19 +231,18 @@ def __init__(self, positions, *, conductivity=0.3, method='psa',
         if voltages is None:
             voltages = list()
 
-        _validate_type(times, list, 'times')
```

Comment:
Can't these type checks remain and not alter the behavior?

Reply:
I was thinking we could rely on `numpy` to throw an error if it's not the right type for `np.array` ...